DDDEastAnglia/Views/Account/_ExternalLoginsPartial.cshtml

```diff
@@ -0,0 +1,49 @@
+﻿@model IEnumerable<DDDEastAnglia.Models.LoginMethodViewModel>
+
+@if (Model.Any())
+{
+    foreach (var loginMethod in Model)
+    {
+        <div style="margin-bottom: 10px;">
```

Comment:
It's a coding standards/code hygiene thing. It's much harder to keep inline styles clean and maintainable than CSS classes. The added benefit of putting it in a class is that it allows us to give the style a meaningful (i.e. semantic) name.